src/solidity/types/Address.js

```diff
@@ -12,4 +12,18 @@ Address.prototype.decodeFromStorage = function (location, storageContent) {
   return '0x' + value.toUpperCase()
 }
 
+Address.prototype.decodeLocals = function (stackHeight, stack, memory) {
+  if (stack.length - 1 < stackHeight) {
+    return '0x0000000000000000000000000000000000000000'
+  } else {
+    return '0x' + util.extractHexByteSlice(stack[stack.length - 1 - stackHeight], this.storageBytes, 0)
+  }
+}
+
+Address.prototype.decodeFromMemory = function (offset, memory) {
+  var value = memory.substr(offset, 64)
```

Comment:
`2*offset` should be here, i.e. the paramater should not be a hex string offset but a byte offset.